app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockViewModel.kt

```diff
@@ -152,6 +162,32 @@ class SetupUnlockViewModel @Inject constructor(
             }
         }
     }
+
+    private fun handleReceiveCreateCipherResult(
+        action: SetupUnlockAction.Internal.ReceiveCreateCipherResult,
+    ) {
+        when (val cipher = action.cipher) {
```

Comment:
Could this just be a `let`:
```kotlin
action
    .cipher
    ?.let {
        sendEvent(
            SetupUnlockEvent.ShowBiometricsPrompt(
                // Generate a new key in case the previous one was invalidated
                cipher = cipher,
            ),
        )
    }
    ?: run {
        mutableStateFlow.update {
            it.copy(
                dialogState = SetupUnlockState.DialogState.Error(
                    title = R.string.an_error_has_occurred.asText(),
                    message = R.string.generic_error_message.asText(),
                ),
            )
        }
    }
```